packages/styled-components/src/sheet/GroupedTag.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import type { Tag } from './Tag';
+
+/** Group-aware Tag that sorts rules by indices */
+export interface GroupedTag {
+  constructor(tag: Tag): void;
+  insertRules(group: number, rules: string[]): void;
+  clearGroup(group: number): void;
+  getGroup(group: number): string;
+  length: number;
+}
+
+/** Create a GroupedTag with an underlying Tag implementation */
+export const makeGroupedTag = (tag: Tag): GroupedTag => {
+  return new DefaultGroupedTag(tag);
+};
+
+const BASE_SIZE = 1 << 8;
```

Comment:
I think this is more bytes than just writing 256

Reply:
Probably, it was just so that I can play around with the number quicker. I'll hardcode it to 256 :+1:

Reply:
The minifier seems to compress it to `256` automatically